fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -58,71 +61,36 @@ public TypeRdfContext(final FedoraResource resource,
 
         //include rdf:type for primaryType, mixins, and their supertypes
         concatRdfTypes();
+        if (resource() instanceof NonRdfSource) {
+            // gather versionability info from the parent
+            if (resource().getNode().getParent().isNodeType("mix:versionable")) {
```

Comment:
Previous to: https://jira.duraspace.org/browse/FCREPO-1411
Properties were being stored on the NonRdfSourceDescription... which can be versionable. But since we are now storing the properties on the NonRdfSource itself, we should not care if the description is versionable. 
We should care if the NonRdfSource is versionable, no?